duckduckgo/Android
# Clean up DBs when clear personal data action executed

Task/Issue URL: https://app.asana.com/0/inbox/1125189844075764/1199685456065107/1200036478062018
Tech Design URL: 
CC: 

**Description**:
This PR adds an extra clean up process when the fire button is used to make sure the deleted data is indeed deleted. We do apply this new change to the cookies db and http_auth.db.

**Steps to test this PR**:
This PR should be tested against all APIs from 21 to 30

1. Launch the app.
1. Go to `https://samesitetest.com/cookies/set`which will store cookies.
1. Go to `https://httpbin.org/basic-auth/u/pw` and enter username=u password=pw.
1. Use the fire button
1. After the app has restarted open the device file explorer
1. Go to `data/data/com.duckduckgo.mobile.android.debug`
1. In the `app_webview` or `app_webview/Default` folder (depending on the API), right click on the `Cookies` DB and click synchronize. After that right click again and save the file in your Desktop (or wherever you want).
1. In the `databases` folder, right click on the `http_auth.db` file and click synchronize. After that right click again and save the file in your Desktop (or wherever you want).

Check the following:
1. In the `databases` folder, the file `http_auth.db-journal` should have been deleted.
1. Open the `Cookies DB` downloaded with a text editor and make sure you cannot find any traces from `samesite`
1. Open the `http_auth.db` downloaded with a text editor and search for `httpbin`. You should not find anything.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
